repository/src/main/java/org/alfresco/repo/security/authentication/identityservice/IdentityServiceConfig.java

```diff
@@ -146,10 +176,7 @@ public String getClientSecret()
 
     public String getIssuerUrl()
     {
-        return UriComponentsBuilder.fromUriString(getAuthServerUrl())
-                                   .pathSegment(REALMS, getRealm())
-                                   .build()
-                                   .toString();
+        return issuerUrl;

```

Comment:
Can you clarify how this is not going to be a breaking change?

Reply:
Before the changes the Issuer URL was auto-generated - now we can configure the Issuer URL. That logic related to a realm has been moved to the `getAuthServerUrl()` method. Now we use authServerUrl to create the OIDC Discovery Endpoint (See: https://github.com/Alfresco/alfresco-community-repo/pull/2426/files/bf3dfc382cbd68088ee76627db8260bd13798315#diff-fdc9f584f86648fc72e778dca91ae854f9340b996d7837e3b441578c0232b8e4R520)
The only case where a compatibility would not be achived has been fixed -> https://github.com/Alfresco/alfresco-community-repo/pull/2426/commits/ba98794eda5280d7b90ac091bb30f75dfdf8df67
Anyway, that `orElseGet` part should not be executed because we validate the OIDC Discovery document endpoints and the Issuer URL is required.